pipenv/cli/command.py

```diff
@@ -416,21 +416,41 @@ def run(state, command, args):
     default=False,
     help="Given a code path, show potentially unused dependencies.",
 )
+@option(
+    "--db",
+    nargs=1,
+    default=False,
+    help="Path to a local vulnerability database. Default: empty",
+)
 @option(
     "--ignore",
     "-i",
     multiple=True,
     help="Ignore specified vulnerability during safety checks.",
 )
+@option(
+    "--output",
+    type=Choice(["default", "json", "full-report", "bare"]),
+    default="default",
+    help="Translates to --json, --ful-report or --bare from safety check",
+)
+@option(
+    "--squelch",
+    is_flag=True,
+    help="Squelch stdout except vulnerability report."
+)
 @common_options
 @system_option
 @argument("args", nargs=-1)
 @pass_state
 def check(
     state,
     unused=False,
+    db=False,
     style=False,
     ignore=None,
+    output="default",
+    squelch=False,
```

Comment:
Seems this flag works similar to `--verbose`?

Reply:
Correct.. But Pipenv always has their STDOUT it seems so i made a "squelch" to hide those outputs.. It makes it easier to use Pipenv in a CI/CD pipeline if you have raw json output.

Reply:
Does Pipenv have something for that? if so I can remove.

Reply:
@techalchemy

Reply:
I'd favor `quiet` here over `squelch` which is what I believe @frostming is referring to.  I don't mind too much about the output flag, but we do use `--json` for `pipenv graph` (this command has other options and if you intend to respect them I can see this being appropriate)

Reply:
I agree, at least coming from a non-English-native background. I can easily imagine my fellow developers having trouble understanding what `squelch` means (let along spell it correctly).

Reply:
That said, maybe `quiet` is not the best choice either (I’d expect it to output next to nothing). Maybe something like `--raw` would be both more understandable and still semantically good enough?